config/locales/en/steps.yml

```diff
@@ -326,7 +326,18 @@ en:
       confirmation:
         show:
           page_title: Application submitted
-          view_application: View completed application
+          heading: 
+            initial: Application completed
```

Comment:
Noted that as this is a shared component this is also changing for the live journey where it was "Application Complete" before - fine by me but thought I would highlight that 👍

Reply:
Would it be ok to revert back to 'Complete'?
Unless there was content review to say otherwise, my preference would be to not change it. 
I take it you wanted 'Evidence submitted' and 'Application complete' to be the same tense?

Reply:
Fantastic spot. Thank you both!